### ISSUE TITLE
Source files for sourcemaps not copied on Linux

### ISSUE THREAD

**Issue by JdeH** [JdeH/Transcrypt#43]
For some reason currently the sourcefiles that the maps refer to are not copied on Linux.

**Comment by JdeH** [comment]
This was only a matter of setting proper access rights.

**Comment by shaolo1** [comment]
When I install via 'sudo python3 setup.py install' and then try to compile my code via 'transcrypt -b -n somefile.py' I get permission errors about sourcemaps.
I worked around this by installing via  'sudo python3 setup.py develop'

**Comment by JdeH** [comment]
You're right about this.
I am not completely sure what's causing this, but I think it's the fact that some sourcemaps come with the installation, which are than only overwriteable with administrator rights.
I'll mark this as a bug and see what I can do to change it, e.g. by deleting the maps before creating the installation.
The same holds for some .js files produced in a build, but it's hard to delete the right ones systematically since modules can also be 'js only'.
As a first step I'll mark this as a bug!

**Comment by JdeH** [comment]
I've delved into this. Transcrypt needs write access to it's installation, even if the map files aren't there. There are several ways to achieve this. You're workaround is one of them, chmod 777 the installation dir is another (I don't say that's right, I just say it's possible...).

But actually the  neatest way to solve this for the average user, is to use virtualenv.
- It will allow you to have distinct setups for distinct projects
- It will install Transcrypt under a user account, solving the access problems

I've tested it and it works under Windows and Linux.
As an alternative to having one env for each  Transcrypt project one could have one env for all transcrypt projects, saving some diskspace. But separating them makes it possible to use different Transcrypt versions alongside, which is probably a good thing.

When using Numscrypt, one would need numpy as well. In that case using the MiniConda "installer" from a user account is the best way to go.

Let me know if using virtualenv works for you.
